setup.py

```diff
@@ -321,7 +321,7 @@ def run(self) -> None:
                                   f"{ex}\nPlease close all AP instances and delete manually.")
 
         # regular cx build
-        self.buildtime = datetime.datetime.utcnow()
+        self.buildtime = datetime.datetime.now(datetime.UTC)
```

Comment:
```suggestion
        self.buildtime = datetime.datetime.now(datetime.timezone.UTC)
```
`datetime.UTC` was added in 3.11, this should be 3.10 compatible